<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cac2X2m</string>
  </property>
  <widget class="QCheckBox" name="voiceCheck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Add CSS voice</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="codeEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>28</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Char code:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Unique ID:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="guidComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>68</y>
     <width>195</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MD5(name+race)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MD5 (name+race+Steam ID of save)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MD5(name)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Random</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="codeButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>28</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate random</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c0Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Costume</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c1Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Costume</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c2Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Costume</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c3Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Costume</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c4Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Costume</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c5Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Costume</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c6Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Costume</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c7Edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Costume</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
